### ISSUE TITLE
Unable to build for web after upgrading to shorebird_code_push 2.0.0

### ISSUE THREAD

**Issue by bgetsug** [shorebirdtech/updater#239]
I cannot build for the web with `shorebird_code_push` version 2.0.0:

```
../../../../.pub-cache/hosted/pub.dev/shorebird_code_push-2.0.0/lib/src/updater.dart:1:8: Error: Dart library 'dart:ffi' is not available on this platform.
import 'dart:ffi' as ffi;
       ^
Context: The unavailable library 'dart:ffi' is imported through these packages:

    web_entrypoint.dart => package:emce_mobile => package:shorebird_code_push => dart:ffi
    web_entrypoint.dart => package:emce_mobile => package:shorebird_code_push => package:ffi => dart:ffi
    ...
```

(Perhaps a regression of #57?)

Version 1.1.6 still works 😅

**Comment by eseidel** [comment]
Thanks Ben.  Just an oversight on our part.  We can get a quick fix out in the morning (and probably update our bots to make sure they build with Web as a test).

**Comment by longtn-imt** [comment]
Min sdk is hight
Can downgrade to `sdk: ^3.5.0`
Flutter 3.24.0

**Comment by bryanoltman** [comment]
@longtn-imt This seems unrelated to the issue being discussed above. Version 2.0 of the updater only supports Flutter 3.24.5 and above. Version 1.0 will still work for older versions of Flutter.

**Comment by eseidel** [comment]
This is resolved in shorebird_code_push 2.0.2.  Thanks for the report!
